### DISCUSSION TITLE
Slow responses

### DISCUSSION THREAD

**Discussion by AThomsen** [microsoft/PubSec-Info-Assistant#442] [Morning (EST) Office Hours (Q&A)]
Our deployment currently takes about 30 second to response to questions. Is there any way to speed this up?
gpt-4-32k in SE-Central (80000 rate limit)

**Comment by ArpitaisAn0maly** [comment]
gpt-4 32K is indeed slower than 3.5 turbo overall. 

But Sometimes the delay is region specific as well. I would try GPT4-Turbo model which is widely available now and also change the region and see if speeds up your response.
